linera-core/src/data_types.rs

```diff
@@ -70,6 +70,8 @@ pub struct ChainInfoQuery {
     pub request_fallback: bool,
     /// Query a value that contains a binary hashed certificate value (e.g. bytecode) required by this chain.
     pub request_hashed_certificate_value: Option<CryptoHash>,
+    /// Query a value that contains a binary blob (e.g. bytecode) required by this chain.
+    pub request_blob: Option<BlobId>,
```

Comment:
By "value" we meant `HashedCertificateValue` here in the first place.
```suggestion
    /// Query a certificate value that contains a binary blob (e.g. bytecode) required by this chain.
    pub request_hashed_certificate_value: Option<CryptoHash>,
    /// Query a binary blob (e.g. bytecode) required by this chain.
    pub request_blob: Option<BlobId>,
```